openshift/openshift/src/main/java/org/arquillian/cube/openshift/impl/client/OpenShiftClientCreator.java

```diff
@@ -24,26 +25,37 @@ public void createClient(@Observes AfterStart afterStart, Configuration conf) {
         CubeOpenShiftConfiguration configuration = (CubeOpenShiftConfiguration) conf;
         System.setProperty("KUBERNETES_TRUST_CERT", "true");
         // override defaults for master and namespace
-        final Config config = new ConfigBuilder()
+        final ConfigBuilder configBuilder = new ConfigBuilder()
```

Comment:
Can you undo the changes of this class? My PR comes with a fix which is a totally different approach by reusing same client instead of instantiating a new one.

Reply:
@lordofthejars I cannot see any parameters for authentication in your PR